cardano-api/src/Cardano/Api/CBOR.hs

```diff
@@ -6,6 +8,11 @@ module Cardano.Api.CBOR
   , publicKeyFromCBOR
   , publicKeyToCBOR
 
+  , decodeShelleyVerificationKey
```

Comment:
I find `'decode` and `encode` a little too imprecise. That is why I named these as `typenameToCBOR` and `typenameFromCBOR` .